src/call-session/index.ts

```diff
@@ -182,6 +212,15 @@ class CallSession extends EventEmitter {
     };
   }
 
+  public async completeWarmTransfer(newSession: CallSession) {
+    const target = newSession.remoteNumber;
+    return await this._transfer(
+      `"${target}@sip.ringcentral.com" <sip:${target}@sip.ringcentral.com;transport=wss?Replaces=${newSession.callId}%3Bto-tag%3D${
+        newSession.remoteTag
+      }%3Bfrom-tag%3D${newSession.localTag}>`,
+    );
+  }
```

Comment:
`warmTransfer(...)` will return `{
      complete: () => Promise<void>;
      cancel: () => Promise<void>;
      newSession: OutboundCallSession;
    }`
    
    You can just invoke the `complete()` method to complete the transfer. 
    I am not sure that we should add this function.

Reply:
Done: https://github.com/ringcentral/ringcentral-web-phone/commit/d99c94a3061f322c2aa7beecc0b3f7311cb094f0